account_bank_statement_import_camt_oca/models/parser.py

```diff
@@ -133,6 +133,8 @@ def parse_entry(self, ns, node):
             transaction = transaction_base.copy()
             self.parse_transaction_details(ns, node, transaction)
             yield transaction
+            if self.env.user.company_id.camt_import_batch:
+                break
```

Comment:
Another difference I've found between our codes is that yours takes the payment reference of the first transaction in the batch and ours uses `Btch/PmtInfId`. I have no idea what is better, but thought I'd mention.